src/components/Tag.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import type { ITag } from '../types/tag'
```

Comment:
tạo file index cho folder types và export file tag.ts ở file index này
thay đổi dòng phía trên sử dụng path alias, nhớ bổ sung path alias ở config nếu chưa có
```suggestion
import type { ITag } from '@types'
```

Reply:
fixed